Tasks/GitHubReleaseV0/main.ts

```diff
@@ -46,9 +46,12 @@ class Main {
                 const githubReleaseAssetInputPatterns = tl.getDelimitedInput(Inputs.assets, Delimiters.newLine);
 
                 if (action === ActionType.create) {
+                    //Get task inputs specific to create release
+                    const tagPattern = tl.getInput(Inputs.tagPattern) || undefined;
```

Comment:
shouldn't there be some validation for this input?

Reply:
Expecting strings here. No validation needed. The attribute is not required too.

Reply:
nit:  I think we can skip `|| undefined` part by passing second argument `true` to tl.getInput. Which indicates optional input.